components/wordPrompt.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import "tailwindcss/tailwind.css";
+
+const WordPrompt = () => {
+  const changeWordHandler = () => {
+    setWords((lastWords) => {
+      if (lastWords.nextWords.length != 0) {
+        return {
+          ...lastWords,
+          currentWord: lastWords.nextWords.shift(),
+        };
+      } else {
+        return {
+          ...lastWords,
+          currentWord: "Out of words!",
+        };
+      }
+    });
+  };
+  const [words, setWords] = useState({
+    currentWord: "Start",
+    nextWords: [],
+  });
+  const fetchWords = () => {
+    fetch("https://random-word-api.herokuapp.com/word?number=10", {
+      method: "GET",
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        setWords((lastWords) => {
+          lastWords.nextWords.push(...data);
+          return { ...lastWords };
+        });
+      })
+      .catch((error) => {
+        console.log(error);
```

Comment:
we shouldn't merge console logs into the main branch, you could just not catch anything for now, if we catch a bug we will fix it

Reply:
Normally, if there's an error, you'd wanna display some error to the user or log it in the database. But this is outside the scope of this PR

Reply:
Okay, got it. I've removed the catch, and I'll handle it in a later issue.